ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogCache.java

```diff
@@ -167,8 +165,27 @@ int size() {
       }
     }
 
-    long sizeInBytes() {
-      return sizeInBytes;
+    long getTotalFileSize() {
+      try(AutoCloseableLock readLock = readLock()) {
+        long size = 0;
+        // TODO(runzhiwang): If there is performance problem, we do not need to re-calculate it every time
+        // because it is only for metric
+        for (LogSegment seg : segments) {
+           size += seg.getTotalFileSize();
+        }
+        return size;
+      }
+    }
```

Comment:
We cannot control how often the metric call this method.  We should cache it or use the original code.  Let's do it separately.